execution_engine/src/storage/trie_store/operations/tests/mod.rs

```diff
@@ -15,6 +15,8 @@ use std::{
     ops::Not,
 };
 
+use backtrace::Backtrace;
+use libc::c_void;
```

Comment:
can I get some info about the choice to attempt bactracing etc for these tests?

Reply:
We wanted to track the number of times `from_bytes` was called when deserializing a value. One way to do this is to track the callers of `from_bytes` and determine if it's one of the functions that we are trying to restrict.